V2/time_tracker/utils/parsers/__init__.py

```diff
@@ -0,0 +1,2 @@
+# flake8: noqa
+from .activity_parser import parse_status_to_number, parse_status_to_string_for_ui
```

Comment:
should be another more generic name no activity_parser maybe status_parser or another name